netto/src/main.rs

```diff
@@ -27,11 +27,11 @@ struct Cli {
     frequency: u64,
 
     /// Bind address or hostname for the web frontend
```

Comment:
Can you change the doc comment to something that reflects the new meaning of the option? Something like
```suggestion
    /// Address of the Grafana Pyroscope backend
```
Same for line 33.

These comments are used by `clap` to generate the help message of the tool's cli, so I think it's important to keep them relevant.
Looks good otherwise!